fs/procfs/fs_procfs.c

```diff
@@ -375,15 +363,14 @@ static int procfs_open(FAR struct file *filep, FAR const char *relpath,
           /* Match found!  Stat using this procfs entry */
 
           DEBUGASSERT(g_procfs_entries[x].ops &&
-              g_procfs_entries[x].ops->open);
+                      g_procfs_entries[x].ops->open);
 
           ret = g_procfs_entries[x].ops->open(filep, relpath, oflags, mode);
-
           if (ret == OK)
             {
               DEBUGASSERT(filep->f_priv);
 
-              ((struct procfs_file_s *) filep->f_priv)->procfsentry =
+              ((struct procfs_file_s *)filep->f_priv)->procfsentry =
```

Comment:
Note for one of next PRs.
```suggestion
              ((FAR struct procfs_file_s *)filep->f_priv)->procfsentry =
```